docker/tendermint-automated/Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM ubuntu:16.04
```

Comment:
you did not like docker image that Tendermint has? 😄

Reply:
No, I do, but I had to point to somewhere on hub.docker.com for tendermint 0.17.1 release and because of the CG_ENABLED issue I couldn't use the alpine one in the tendermint repo. Now that the issue is fixed, I can open a PR on the tendermint repo, that updates that dockerimage. I didn't want to introduce ubuntu there, unless absolutely necessary. I would like to move back to using alpine for docker.